tools/vcd-parsing/get-profile-counts-info.sh

```diff
@@ -0,0 +1,44 @@
+# Wrapper script for running TDCC, running simulation, and obtaining cycle counts information
```

Comment:
Please create a top-level README.md file for this folder explaining what the scripts are here for.